BUG: Docker build CMake error

### Make sure you've done the following:

- [X] I have checked that this issue has not already been reported.

- [X] I have validated that this issue is not a syntax error of either MySQL or SQLite.

- [X] I have pulled the latest version of the main branch of DarkflameServer and have confirmed that the issue exists there.


### DarkflameServer Version

6c97ea8

### Issue Description

After updating to the latest `main` (to the version listed above), I tried to run `docker compose build` (part of what I normally run to update the server) and this error occurred:

```
 => ERROR [server_darkflame build 22/23] RUN echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_buil  2.1s
------
 > [server_darkflame build 22/23] RUN echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_build &&     sed -i -e "s/171022/171022/g" ../CMakeVariables.txt &&     cmake .. -DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE &&     make -j 2:
#40 0.594 Build server
#40 0.806 -- The C compiler identification is GNU 11.2.0
#40 0.978 -- The CXX compiler identification is GNU 11.2.0
#40 0.992 -- Detecting C compiler ABI info
#40 1.205 -- Detecting C compiler ABI info - done
#40 1.251 -- Check for working C compiler: /usr/bin/cc - skipped
#40 1.253 -- Detecting C compile features
#40 1.262 -- Detecting C compile features - done
#40 1.274 -- Detecting CXX compiler ABI info
#40 1.685 -- Detecting CXX compiler ABI info - done
#40 1.710 -- Check for working CXX compiler: /usr/local/bin/c++ - skipped
#40 1.711 -- Detecting CXX compile features
#40 1.713 -- Detecting CXX compile features - done
#40 1.723 -- Variable: PROJECT_VERSION_MAJOR = 1
#40 1.724 -- Variable: PROJECT_VERSION_MINOR = 0
#40 1.724 -- Variable: PROJECT_VERSION_PATCH = 4
#40 1.724 -- Variable: LICENSE = AGPL-3.0
#40 1.725 -- Variable: NET_VERSION = 171022
#40 1.725 -- Variable: __dynamic = 1
#40 1.726 -- Variable: __maria_db_connector_compile_jobs__ = 1
#40 1.726 -- Version: 1.0.4
#40 1.727 Moved authconfig.ini to project binary directory
#40 1.728 Moved chatconfig.ini to project binary directory
#40 1.730 Moved worldconfig.ini to project binary directory
#40 1.731 Moved masterconfig.ini to project binary directory
#40 1.732 Moved blacklist.dcf to project binary directory
#40 1.816 -- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11")
#40 1.861 CMake Error at CMakeLists.txt:211 (add_subdirectory):
#40 1.861   add_subdirectory given source "dNavigation" which is not an existing
#40 1.861   directory.
#40 1.861
#40 1.862
#40 1.874 -- Configuring incomplete, errors occurred!
#40 1.874 See also "/build/cmake_build/CMakeFiles/CMakeOutput.log".
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_build &&     sed -i -e "s/171022/${BUILD_VERSION}/g" ../CMakeVariables.txt &&     cmake .. -DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE &&     make -j $BUILD_THREADS]: exit code: 1
```

### Reproduction Steps

Using a `docker compose` setup on Linux:

1. Checkout 6c97ea8
2. Run `git submodule update --init --recursive` (as instructed [here](https://github.com/DarkflameUniverse/DarkflameServer#updating-your-build))
3. Run `docker compose build`

### Expected Behavior

The containers should build successfully.

### Environment

Ubuntu 20.04.4 LTS, on an AWS EC2 t3a.medium instance

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -21,6 +21,7 @@ COPY dPhysics/ /build/dPhysics
 COPY dScripts/ /build/dScripts
 COPY dWorldServer/ /build/dWorldServer
 COPY dZoneManager/ /build/dZoneManager
+COPY dNavigation/ /build/dNavigation
 COPY migrations/ /build/migrations
 COPY resources/ /build/resources
 COPY thirdparty/ /build/thirdparty

